frontend/tests/unit/services/user-service-test.js

```diff
@@ -0,0 +1,12 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Service | user-service', function(hooks) {
+  setupTest(hooks);
+
+  // Replace this with your real tests.
+  test('it exists', function(assert) {
+    let service = this.owner.lookup('service:user-service');
+    assert.ok(service);
+  });
```

Comment:
Either put some real tests here or delete the whole file.
If the service doesn't have any logic, a unit test seems pretty useless :)